app/src/main/java/com/karumi/screenshot/ui/view/SuperHeroDetailActivity.java

```diff
@@ -22,68 +22,75 @@
-  }
-
-  public static void open(Context context, String superHeroName) {
-    Intent intent = new Intent(context, SuperHeroDetailActivity.class);
-    intent.putExtra(SUPER_HERO_NAME_KEY, superHeroName);
-    context.startActivity(intent);
-  }
-
-  @Override protected void initializeToolbar() {
-    super.initializeToolbar();
-    String superHeroName = getSuperHeroName();
-    setTitle(superHeroName);
-  }
-
-  private void initializeDagger() {
-    SuperHeroesApplication app = (SuperHeroesApplication) getApplication();
-    app.getMainComponent().inject(this);
-  }
-
-  private void initializePresenter() {
-    presenter.setView(this);
-    String name = getSuperHeroName();
-    presenter.setName(name);
-    presenter.initialize();
-  }
-
-  private String getSuperHeroName() {
-    return getIntent().getExtras().getString(SUPER_HERO_NAME_KEY);
-  }
+    private static final String SUPER_HERO_NAME_KEY = "super_hero_name_key";
+
+    @Inject SuperHeroDetailPresenter presenter;
+
+    @BindView(R.id.iv_super_hero_photo) ImageView superHeroPhotoImageView;
+    @BindView(R.id.tv_super_hero_name) TextView superHeroNameTextView;
+    @BindView(R.id.tv_super_hero_description) TextView superHeroDescriptionTextView;
+    @BindView(R.id.iv_avengers_badge) View avengersBadgeView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        initializeDagger();
+        initializePresenter();
+    }
+
+    @Override
+    public int getLayoutId() {
+        return R.layout.super_hero_detail_activity;
+    }
+
+    @Override
+    public void showSuperHero(SuperHero superHero) {
+        Picasso.get().load(superHero.getPhoto()).fit().centerCrop().into(superHeroPhotoImageView);
+        superHeroNameTextView.setText(superHero.getName());
+        superHeroDescriptionTextView.setText(superHero.getDescription());
+        int avengersBadgeVisibility = superHero.isAvenger() ? View.VISIBLE : View.GONE;
+        avengersBadgeView.setVisibility(avengersBadgeVisibility);
+    }
+
+    public static void open(Context context, String superHeroName) {
+        Intent intent = new Intent(context, SuperHeroDetailActivity.class);
+        intent.putExtra(SUPER_HERO_NAME_KEY, superHeroName);
+        context.startActivity(intent);
+    }
+
+    @Override
+    protected void initializeToolbar() {
+        super.initializeToolbar();
+        String superHeroName = getSuperHeroName();
+        setTitle(superHeroName);
+    }
+
+    private void initializeDagger() {
+        SuperHeroesApplication app = (SuperHeroesApplication) getApplication();
+        app.getMainComponent().inject(this);
+    }
+
+    private void initializePresenter() {
+        presenter.setView(this);
+        String name = getSuperHeroName();
+        presenter.setName(name);
+        presenter.initialize();
+    }
+
+    private String getSuperHeroName() {
+        return getIntent().getExtras().getString(SUPER_HERO_NAME_KEY);
+    }
```

Comment:
You've not configured the code format properly and this is going to break the build. [This file](https://github.com/Karumi/KataScreenshotAndroid/blob/master/config/checkstyle.xml) explains how to configure your IDE to format the code automatically, could you apply it?